holmscan/constants.py

```diff
@@ -0,0 +1,3 @@
+# -*- coding: utf-8 -*-
+
+SCAN_STATUS_CHOICES = ["completed", "running"]
```

Comment:
When doing choices constants, even the choices should be made into constants.
```
SCAN_STATUS_COMPLETED = 'completed'
SCAN_STATUS_RUNNING = 'running'

SCAN_STATUS_CHOICES = [
    SCAN_STATUS_COMPLETED,
    SCAN_STATUS_RUNNING,
]
```

Reply:
I don't get the added benefit of this.